lib/ansible/modules/cloud/vultr/vultr_server.py

```diff
@@ -54,6 +54,12 @@
     description:
       - Whether to enable private networking or not.
     type: bool
+  network_id:
```

Comment:
Based on the documentation https://www.vultr.com/api/#server_create

> NETWORKID array (optional) List of private networks to attach to this server.  Use either this field or enable_private_network, not both

So I would make it a lists. Also for naming/user experience, I'd rename it networks where user can input a lists of networks name (easier to read/understand). Later a `get_networks()` method could be used to translate from name to id to pass values in the payload.

Reply:
I am also +1 what @Spredzy suggested.